.github/workflows/zcash-lightwalletd.yml

```diff
@@ -52,7 +50,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3.0.2
         with:
-          repository: adityapk00/lightwalletd
+          repository: zcash/lightwalletd
```

Comment:
Review Note: This switches our tests to use the zcash lightwalletd fork.

(Please close this comment after reading it, so the PR can merge.)